### ISSUE TITLE
Consider if Tiny yolo loss function is wrong

### ISSUE THREAD

**Issue by GiangHLe** [qqwweee/keras-yolo3#638]
https://github.com/qqwweee/keras-yolo3/blob/e6598d13c703029b2686bc2eb8d5c09badf42992/yolo3/model.py#L364

The anchor_mask of tiny_yolo was wrong, right? It should be [[3,4,5],[0,1,2]]

**Comment by coldrappit** [comment]
you are right. It should be [[3,4,5],[0,1,2]]

**Comment by robisen1** [comment]
Did the author react to this? Does he share access with anyone else so they can update code and resolve issues?

**Comment by GiangHLe** [comment]
no, you can folk it then pull request. However, I haven't seen the author for a long time, I think he dropped this repo and this is a simple bug that we can easy to realize and fix it by ourselves.

**Comment by robisen1** [comment]
Yes I understand that it can be forked. A good number have. The problem is they are all different changes made to the code base. A central repo would be great. That being said it is what it is.
